apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -890,13 +890,13 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
 
         if feedback_type in FIRST_WRONG_TYPES:
             for block in student_answer[:num_initial_correct]:
-                block["badge_type"] = "badge-success"
+                block["badge_type"] = "text-bg-success"
                 block["icon"] = "fa-check"
                 block["distractor_feedback"] = ""
                 block["ordering_feedback"] = ""
 
             if first_wrong is not None:
-                student_answer[first_wrong]["badge_type"] = "badge-danger"
+                student_answer[first_wrong]["badge_type"] = "text-bg-danger"
```

Comment:
I just pulled this to check the UI for order block, looks good :+1: 

I'm not going to approve because I didn't review/test the rest though.